RUFAS/biophysical/animal/herd_factory.py

```diff
@@ -541,8 +541,14 @@ def initialize_herd(self) -> None:
         MilkProduction.set_milk_quality(
             AnimalConfig.milk_fat_percent, AnimalConfig.true_protein_percent, AnimalModuleConstants.MILK_LACTOSE
         )
-
+        # self.init_herd = True
         if self.init_herd:
+            if AnimalConfig.semen_type == "sexed":
+                om.add_warning(
+                    "Longer herd generation runtime.",
```

Comment:
```suggestion
                    "Longer herd generation runtime",
```
There's a period in the standard log/warning/message name so adding it again makes it look off:
```
[07-May-2025_Wed_16-49-13.953860][WARNING][default] Longer herd generation runtime.. Herd initialized with sexed semen will result in significantly longer runtime.
```